### PR TITLE
#2235 Stream API could not update from or to field to empty

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2235)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala [hunk 1]
```diff
@@ -188,8 +188,10 @@ private[configurator] object StreamRoute {
           settings = req.settings ++
             Map(
               DefaultConfigs.NAME_DEFINITION.key() -> JsString(key.name),
-              DefaultConfigs.FROM_TOPICS_DEFINITION.key() -> JsArray(req.from.map(JsString(_)).toVector),
-              DefaultConfigs.TO_TOPICS_DEFINITION.key() -> JsArray(req.to.map(JsString(_)).toVector),
+              DefaultConfigs.FROM_TOPICS_DEFINITION.key() -> JsArray(
```
Comment (chia7712) [inline ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala]: the update without "from" will change the previous setting to "Seq.empy". It is a bug, right?

Comment (saivirtue) [reply ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala]: Nope. This the case "use UPDATE as POST". If user request without "from", we generate Set.empty to response.

Comment (chia7712) [reply ohara-configurator/src/main/scala/com/island/ohara/configurator/route/StreamRoute.scala]: got it :)

**Comment by saivirtue** [comment]
retry client

**Pull request merged**